### DISCUSSION TITLE
OpenTelemetry: How to suppress the logs emitting from opentelemetry-javaagent

### DISCUSSION THREAD

**Discussion by sivaponting** [open-telemetry/opentelemetry-java-instrumentation#11186] [Q&A]
Here is the use-case,
We are using opentelemetry-instrumentation to monitor our application. We have extentions to capture the Apache-Spark logs.
Now it's capturing both the application logs as well as spark logs. 

**otel.logs.exporter=otlp**
otel.traces.exporter=otlp
otel.metrics.exporter=otlp

Is thr any mechanism to suppress only application logs?

**Comment by jkwatson** [comment]
moving this question over to the instrumentation repo

**Comment by krishnaexplore** [comment]
@sivaponting 

-> use otel.logs.exporter=none

**Reply by sivaponting** [reply]
I don't want to suppress the entire log. Want to suppress only particular package name or scope name

**Reply by trask** [reply]
there is recently added specification to support disabling loggers per scope name: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/sdk.md#loggerconfigurator

but it still needs to be added to declarative configuration and to Java SDK

in the meantime, you would need to write an [agent extension](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/examples/extension)

cc @jack-berg
